src/kaleidoscope/driver/led/Base.h

```diff
@@ -65,14 +65,14 @@ class Base {
     return pgm_read_byte(&_LEDDriverProps::key_led_map[key_offset]);
   }
 
-  static class LEDs {
+  class LEDs__ {
```

Comment:
Is there a reason for using `LEDs__` instead of `LEDs_` here? (Consequently, if there isn't, then `leds__` below should have one underscore removed too).

Reply:
I've been burned before by having types and members with same name in one class. That's why I avoid it where I can. As we have the convention to add a trailing underscore to non-static member data entities, I use two underscore in such cases.

I removed the trailing underscores of `leds__`, though.

Reply:
Mhm, that makes sense. I'll wait for @obra's opinion before merging, but I'm satisfied with this reasoning. Thanks!

For context, the class itself should not be directly needed by anything. It's primary use is for `Kaleidoscope.device().LEDs()` to return an instance of, of which we'll immediately call a method of: `Kaleidoscope.device().LEDs().all()`, etc. So having two underscores in the name is something hardly anyone will ever see. Neither end-users nor plugin authors will need to care about this particular class' name.